frontend/src/components/pages/spells/SpellListItem.js

```diff
@@ -0,0 +1,52 @@
+import { Card, CardActions, CardContent, CardHeader, CardMedia, Grid, Typography } from "@mui/material";
+import Image from "next/image";
+
+import { faWandSparkles } from "@fortawesome/free-solid-svg-icons";
+
+import Link from '../../Link'
+import ListItemBox from '../../ui/ListItemBox'
+
+
+const SpellListItem = ({ spell }) => {
+  const { name, slug, effect, hand, incantation, category, light, creator, image, wiki } = spell.attributes;
```

Comment:
Please remove unused attributes if they are not used. Here for example effect, hand, category, light, creator and wiki.